### ISSUE TITLE
busybox after initramfs-tools update

### ISSUE THREAD

**Issue by hakuna-m** [hakuna-m/wubiuefi#5]
I received a bug report that some users got a busybox prompt after initramfs updates on Ubuntu 16.04.

I updated initramfs-tools from 0.122ubuntu**6** to 0.122ubuntu**7** and I could reproduce that issue. It was possible to change ro -> rw in Grub as a workaround.

Meanwhile I found a solution. If you type
`sudo update-initramfs -u`   
the initramfs patch is copied to the current initrd.img file and all things work as before.

It seems that in some cases an additional update is needed. But IMHO mainly it is an issue of versions under development. It doesn't affect a fresh install or an upgrade because every kernel update also triggers an additional initramfs update.

**Comment by hakuna-m** [comment]
I will fix it because currently initramfs-tools seems less stable than in the past ( [see publishing history]( https://launchpad.net/ubuntu/+source/initramfs-tools/+publishinghistory) ) and I don't expect that [fix of noorez kassam](https://code.launchpad.net/~noorez-kassam/ubuntu/utopic/initramfs-tools/fix-for-1317437) will be released in the near future.

**Comment by hakuna-m** [comment]
I changed  **/etc/initramfs-tools/hooks/loop-remount**. Now the script copies the patched file **/usr/share/initramfs-tools/scripts/local** to **scripts/local** of initramfs, if needed. This is necessary to be sure that the patch is also applied to initramfs.

Additionally, I inserted a header for prereqs for future use.

It is not necessary to do a new install for using the changed loop-remount. Just download the changed file, set it to executable and run update-initramfs.
```
sudo wget https://raw.githubusercontent.com/hakuna-m/wubiuefi/master/data/custom-installation/patch/loop-remount -O /etc/initramfs-tools/hooks/loop-remount
sudo chmod +x /etc/initramfs-tools/hooks/loop-remount
sudo update-initramfs -u
```
